spec/features/foods_spec.rb

```diff
@@ -1,18 +1,73 @@
 require "rails_helper"
 RSpec.feature "Foods", type: :feature do
-  scenario "the food index page shows all the food" do
-    Food.create(
+  before(:each) do
+  @user = User.create(first_name: "Toyosi",
+                      last_name: "Famakinde",
+                      email: "toyosif@yahoo.com",
+                      password: "jeffrules",
+                      password_confirmation: "jeffrules",
+                      role: "admin")
+  Category.create(title: "Entree", description: "first meal")
+  @F1 = Food.create(
       name: "Spaghetti",
       description: "Best spaghetti ever",
-      price: 1500.00
-    )
-    Food.create(
-      name: "Jollof rice",
-      description: "Hot spicy jollof ever",
-      price: 1600.00
-    )
+      price: 1500.00 )
+  visit root_path
+
+  click_link('Log in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.